### PR TITLE
Improve mapillary correlation

### PR DESCRIPTION
**Pull request by zlavergne**
Improving task sizes by increasing the time correlation from 1 minute to 10 minutes.

### CODE REVIEW

File: server/services/mapillary_service.py [hunk 1]
```diff
@@ -149,7 +149,7 @@ def getMapillarySequences(parameters: dict):
             # if props:
             #     tasks.append(geojson.Feature(geometry=feature['geometry'], properties={'mapillary': props}))
 
-            geom = linemerge([shape(t['geometry']) for t in task])
+            geom = linemerge([shape(t['geometry']) for t in task]).simplify(0.2)
```
Comment (tsmock) [inline server/services/mapillary_service.py]: I thought you said that simplify had bad effects?

**Comment by zlavergne** [comment]
arg. I left the simplify test in there @taylorsmock. rebasing...

File: server/services/mapillary_service.py [hunk 2]
```diff
@@ -118,7 +118,7 @@ def getMapillarySequences(parameters: dict):
             task = []
             task.append(feature)
 
-            for otherFeature in [f for f in features if (abs((t1 - parser.parse(f['properties']['captured_at'])).total_seconds()) <= 60)]:
+            for otherFeature in [f for f in features if (abs((t1 - parser.parse(f['properties']['captured_at'])).total_seconds()) <= 600)]:
```
Comment (tsmock) [inline server/services/mapillary_service.py]: Looks good to me (although I think admin/user configurable would be better).

**Review by tsmock — CHANGES_REQUESTED** [review]

**Review by tsmock — APPROVED** [review]

**Review by tsmock — APPROVED** [review]
Looks good to me.

File: server/config.py [hunk 3]
```diff
@@ -56,6 +56,7 @@ class EnvironmentConfig:
 
     # Image sources
     MAPILLARY_API = {'base': 'https://a.mapillary.com/v3/', 'clientId': os.getenv('MAPILLARY_API_KEY', None)}
+    MAPILLARY_TIME_CORRELATION = os.getenv('MAPILLARY_TIME_CORRELATION', 60)
```
Comment (tsmock) [inline server/config.py:59]: Just a note, you want the default to be 60, while we can set it to be 600 in the server config.

Right?

**Pull request merged**